schema/Core/ActionSchedule.entityType.php

```diff
@@ -152,7 +152,7 @@
     ],
     'start_action_date' => [
       'title' => ts('Start Action Date'),
-      'sql_type' => 'varchar(64)',
+      'sql_type' => 'varchar(255)',
```

Comment:
This column gets used for SearchKit-generated field aliases, which can get quite lengthy. 255 will _probably_ be long enough, but why chance it? I was thinking something like 2048.

Reply:
Good idea. I've updated it.